### PR TITLE
[PAN-2810] Privacy group management examples

### PR DESCRIPTION
**Pull request by Puneetha17**
Add functions to enable privacy group management with examples.

### CODE REVIEW

File: test/integration/privacyGroupManagement.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  t.test("can find privacy group", async st => {
+    const result1 = await findGroup();
+
+    st.equal(RESULT, result1);
+
+    st.end();d
+  });
+
+  t.test("can delete privacy group", async st => {
```
Comment (ekellstrand) [inline test/integration/privacyGroupManagement.test.js]: We should have a bunch more testing around all the negative test cases.

Comment (Puneetha17) [reply test/integration/privacyGroupManagement.test.js]: Have added a few more tests now. Though they dont all pass, waiting for all PRs in Pantheon to be merged.
